app/src/main/java/rick_and_morty/ui/characters/composables/CharacterList.kt

```diff
@@ -0,0 +1,84 @@
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.collectAsState
+import androidx.compose.ui.unit.dp
+import androidx.lifecycle.viewmodel.compose.viewModel
+import rick_and_morty.ui.characters.CharactersViewModel
+
+@Composable
+fun CharacterList() {
+    val charactersViewModel = viewModel(modelClass = CharactersViewModel::class.java)
+
+    var page = 1;
+
+    charactersViewModel.getCharacters(page)
```

Comment:
This could be called in the `init {...}` block of the `charactersViewModel` so it would be triggered as soon as the view model is created. We could save a function call here.

Reply:
Done